airbyte-integrations/connectors/destination-chroma/destination_chroma/config.py

```diff
@@ -0,0 +1,96 @@
+#
+# Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+#
+
+from typing import Literal, Union
+
+import dpath.util
+from airbyte_cdk.destinations.vector_db_based.config import (
+    CohereEmbeddingConfigModel,
+    FakeEmbeddingConfigModel,
+    FromFieldEmbeddingConfigModel,
+    OpenAIEmbeddingConfigModel,
+    ProcessingConfigModel,
+)
+from airbyte_cdk.utils.spec_schema_transformations import resolve_refs
+from pydantic import BaseModel, Field
+
+
+class HttpMode(BaseModel):
+    mode: Literal["http_client"] = Field("http_client", const=True)
+    host: str = Field(..., title="Host", description="The URL to the chromadb instance", order=0)
+    port: str = Field(..., title="Port", description="The port to the chromadb instance", order=1)
```

Comment:
You can do `port: int` and save yourself the casting (also makes it easier for the user because they won't be able to type something wrong)